airgun/widgets.py

```diff
@@ -1062,6 +1063,16 @@ def read(self):
         do_not_read_this_widget()
 
 
+class ConfirmationBox(ConfirmationDialog):
```

Comment:
I would personally keep the name as Pf4ConfirmationDialog kind of pattern, So Whenever we would like to migrate completely to PF4 then it will be pretty easy to identify it. @latran @jyejare and @JacobCallahan  wanted to know thoughts about it

Reply:
@jameerpathan111 can you please also know where are these are used/added in Satellite 7.0? add supporting code.

Reply:
okay, I will try to find where we need to use this new dialog and raise a separate pr for it.

Reply:
@omkarkhatavkar @jameerpathan111 Sorry for my less knowledge here but does satellite 7.0 is the mixture of PF4 and existing PF version? 

if it's only PF4 :
    can we stop prepending PF4 altogether in airgun cause PF4 is default/only and we not to explicitely mention it ? e.g `Button` instead of `PF4Button`?
else:
    Since there are earlier version dialogs as well then we should prepend the PF4 becuase it would be easy for tomorrow to remove version < PF4 widgets/views/entities to remove.

Reply:
By default, we don't have PF4 everywhere in Satellite 7.0. We are migrating from PF3 to PF4. I would agree on the part if the PF4 elements are not used in combination with PF3 elements then make PF4 as default and no need to add prefix 'PF4' in that case.

Reply:
@jyejare yes, not all buttons are PF4 buttons.

Reply:
@omkarkhatavkar changed widget name to `Pf4ConfirmationDialog`.